src/test/java/com/epam/training/student_gregory_kubya/CartPage.java

```diff
@@ -0,0 +1,69 @@
+package com.epam.training.student_gregory_kubya;
+
+
+import java.util.ArrayList;
+import java.util.List;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindAll;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class CartPage {
+
+  private final WebDriver driver;
+  private static final String PAGE_URL = "https://fiftyoutlet.com/es/es/cart";
+
+
+  @FindAll({@FindBy(css = "div.name.product-name")})
+  List<WebElement> cartNameList;
+  @FindAll({@FindBy(css = "div.sku.product-ref")})
```

Comment:
you should locator to the value element, i.e:
div[class='sku product-ref'] span[class='value']